reduxApp2/dev/js/components/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import FillDetails from '../containers/fill-details';
+import UserDetails from '../containers/user-details';
+require('../../scss/style.scss');
+
+
+const App = () => (
+  <div>
+    <h2>Fill your Details</h2>
+    <FillDetails />
+    <hr />
```

Comment:
@VARTIKASINGH1234 what is the hr here.

Reply:
<hr /> is to add a line for view purpose in UI